shared/actions/profile/proofs.js

```diff
@@ -279,7 +271,7 @@ function * _submitCryptoAddress (action: SubmitBTCAddress | SubmitZcashAddress):
     yield call(cryptocurrencyRegisterAddressRpcPromise, {param: {address, force: true, wantedFamily}})
     yield put(_waitingForResponse(false))
     yield put(_updateProofStatus(true, ProveCommonProofStatus.ok))
-    yield put(navigateAppend(['postProof', 'confirmOrPending'], [profileTab]))
+    yield put(navigateAppend(['confirmOrPending'], [profileTab]))
```

Comment:
don't put paths which we actually can't go to / shouldn't. on mobile since we render mulitple parts of the tree this actually crashes you out cause postProof doesn't allow certain types of things here (like btc)